tuning/data/setup_dataprocessor.py

```diff
@@ -588,6 +595,10 @@ def process_dataargs(
             "Check your data config or ensure split sizes are valid."
         )
     if data_args.do_dataprocessing_only:
+        if odm_config:
+            raise ValueError(
+                "data processing with online data mixing is not currently supported"
```

Comment:
Any reason we do not support data processing only mode for ODM config?

We can just dump the processed datasets and return right?

ODM then has to applied while training too but is there a fundamental problem in compatibility?